spec/jobs/reports/agency_invoice_iaa_supplement_report_spec.rb

```diff
@@ -205,14 +209,26 @@
               iaa_start_date: iaa1_range.begin.to_s,
               iaa_end_date: iaa1_range.end.to_s,
             },
+            {
+              iaa: iaa2_key,
+              ial1_total_auth_count: 0,
+              ial2_total_auth_count: 1,
+              ial1_unique_users: 0,
+              ial2_unique_users: 1,
+              ial1_new_unique_users: 0,
+              ial2_new_unique_users: 1,
+              year_month: inside_iaa2.strftime('%Y%m'),
+              iaa_start_date: iaa2_range.begin.to_s,
+              iaa_end_date: iaa2_range.end.to_s,
+            },
```

Comment:
I *think* this is correct now that this has another section... I think it may have been grouped inaccurately before